apps/frontend/src/app/5_pages/MarketMakingPage/MarketMakingPage.types.ts

```diff
@@ -0,0 +1,13 @@
+export enum PromotionColor {
```

Comment:
You don't need this at all since promo cards in D2 don't use it. Delete the whole file and delete promotion colors config from `AmmLiquidityPool.ts`.